src/lib/NonPartitionedSessionStorage.js

```diff
@@ -113,35 +113,47 @@ class NonPartitionedSessionStorage {
         }
 
         // Create a hash of the user agent to consume less space in the flag cookie than the complete user agent.
+        // We store a hash of the user agent string, to re-run the check on browser update because the partitioning
+        // behavior can change over time.
         const userAgentHash = new Uint8Array(await crypto.subtle.digest(
             'SHA-256',
             Nimiq.BufferUtils.fromAscii(navigator.userAgent), // Don't use Utf8Utils as they're not bundled in iframe.
         ));
         const userAgentHashString = Nimiq.BufferUtils.toBase64Url(userAgentHash).substring(0, 6);
 
-        const otherContextEntryName = NonPartitionedSessionStorage.isIframe()
+        const otherContextEntry = sessionStorage.getItem(NonPartitionedSessionStorage.isIframe()
             ? NonPartitionedSessionStorage.TOP_LEVEL_SESSION_STORAGE_ENTRY
-            : NonPartitionedSessionStorage.IFRAME_SESSION_STORAGE_ENTRY;
-        if (sessionStorage.getItem(otherContextEntryName)) {
+            : NonPartitionedSessionStorage.IFRAME_SESSION_STORAGE_ENTRY);
+        if (otherContextEntry) {
             // The sessionStorage was proven to not be partitioned. Persist this info, such that we also know it in
             // other windows and without having the conditions for successfully running the check, namely Keyguard being
             // opened as top-level and then as iframe, or vice-versa, in the same tab, because sessionStorage always
             // exists per tab only. Note that this check would be inaccurate if the iframe would be embedded into a
             // Keyguard top level window, because no partitioning occurs if the top-level window and the iframe are on
             // the same origin. As the Keyguard is only ever embedded as iframe into the Hub, this check is accurate.
-            // Also note, that in the opposite case, where an entry is not readable in the other context, nothing is
-            // proven because the Keyguard might not have been opened in the other context before in the same tab.
-            // We store a hash of the user agent string, to re-run the check on browser update because the partitioning
-            // behavior can change over time.
-            CookieJar.writeCookie(CookieJar.Cookie.FLAG_SESSION_STORAGE_NOT_PARTITIONED, userAgentHashString);
+            CookieJar.writeCookie(CookieJar.Cookie.FLAG_SESSION_STORAGE_PARTITIONED, `0${userAgentHashString}`);
```

Comment:
Can you make the `0` and `1` prefixes into variables? E.g. an "enum": 
```
const PREFIXES = {
    WHATEVER_0_MEANS: '0',
    WHATEVER_1_MEANS: '1',
}
```

Because I don't understand what they mean.